common/src/main/java/com/wynntils/mc/mixin/ResourceLoadStateTrackerMixin.java

```diff
@@ -17,6 +17,6 @@ public abstract class ResourceLoadStateTrackerMixin {
     private void onResourceManagerReloadPost(CallbackInfo info) {
         // This is the signal that Minecraft has finished loading the initial resources,
         // or a resource pack has been reloaded
-        WynntilsMod.onResourcesFinishedLoading();
```

Comment:
It is possible that the old name was not enough descriptive, but the new name is really no better. This is called after all resources has been (re-)loaded. This can happen multiple time. But the fact that the resources are done loading (which is an async event) is important.